### PR TITLE
Question

### PR DESCRIPTION
**Pull request by sokqa**
# Git workshop 2022-11-15

* created new text file about my favorite programming language

### CODE REVIEW

File: statement.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Python is my favorite language
+I miss it so much
```
Comment (sokqa) [inline statement.txt:2]: I know right

File: statement.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+also Scala is not bad
```
Comment (ArturKovacsCC) [inline statement.txt:4]: I would probably add Rust as well

**Review by ArturKovacsCC — CHANGES_REQUESTED** [review]

File: statement.txt (hunk 2 above)
Comment (sokqa) [reply statement.txt:4]: sorry, fixing it right away

**Review by ArturKovacsCC — APPROVED** [review]
LGTM

**Comment by sokqa** [comment]
Closing workshop

**Pull request closed without merging**